.github/workflows/test.yml

```diff
@@ -3,13 +3,13 @@ name: Tests
 on:
   push:
     branches:
-      - main
       - stable/*
       - release/*
```

Comment:
:question: Out of scope, but I think release branches are `release-` not `release/` :smile: Does this work? If so, can you explain to me why? I feel dumb for not getting it.

Reply:
Whoops, that was my mistake. I'll change that to `release-*`